cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -100,23 +100,23 @@ class HeadNodeRootVolumeSchema(BaseSchema):
         validate=get_field_validator("volume_type"),
         metadata={
             "update_policy": UpdatePolicy(
-                UpdatePolicy.UNSUPPORTED, action_needed=UpdatePolicy.ACTIONS_NEEDED["ebs_volume_update"]
```

Comment:
based on offline conversations from earlier today i think we still want to keep all of these as unsupported

Reply:
correct, done